tests/ballerina-compiler-api-test/src/test/resources/test-src/expected-type/annotation_test.bal

```diff
@@ -0,0 +1,297 @@
+    @CustomAnnotation {
+        info: {},
+        doc: {}
+    }
+    remote function name() {
+
+    }
+
+    @CustomAnnotation {
+        info: {},
+        doc: {}
+    }
+    resource function accessor path() {
+
+    }
+}
+
+@CustomAnnotation {
+    info: {},
+    doc: {}
+}
+type A service object {
+    @CustomAnnotation {
+        info: {},
+        doc: {}
+    }
+    remote function name();
+
+    @CustomAnnotation {
+        info: {},
+        doc: {}
+    }
+    resource function accessor path();
+};
+
+var a = service object {
+    @CustomAnnotation {
+        info: {},
+        doc: {}
+    }
+    int i;
+
+    @CustomAnnotation {
+        info: {},
+        doc: {}
+    }
+    remote function name() {
+
+    }
+
+    @CustomAnnotation {
+        info: {},
+        doc: {}
+    }
+    resource function accessor path() {
+
+    }
+};
+
+function testFunctionWithReturn() returns @CustomAnnotation {info: {}, doc: {}} int => 10;
+
+function testFunctionWithParams(@CustomAnnotation {info: {}, doc: {}} int i,
+        @CustomAnnotation {info: {}, doc: {}} int j = 32,
+        @CustomAnnotation {info: {}, doc: {}} *Info info,
+        @CustomAnnotation {info: {}, doc: {}}
+        anydata... rest) {
+}
+
+type TestRecord record {|
+    @CustomAnnotation {
+        info: {},
+        doc: {}
+    }
+    string value;
+|};
+
+@CustomAnnotation {
+    info: {},
+    doc: {}
+}
+listener TestClass testListener = new;
+
+@CustomAnnotation {
+    info: {},
+    doc: {}
+}
+var testModuleVarDecl = 32;
+
+function testLocalVarDecl() {
+    @CustomAnnotation {
+        info: {},
+        doc: {}
+    }
+    var testLocalVarDecl = 32;
+
+    _ = let @CustomAnnotation {info: {}, doc: {}}
+        var testLetVarDecl = 32
+        in testLetVarDecl + 32;
+}
+
+@CustomAnnotation {
+    info: {},
+    doc: {}
+}
+annotation TestAnnotation on field;
+
+function testWorker() {
+    _ = @CustomAnnotation {info: {}, doc: {}} start testFunction();
+
+    @CustomAnnotation {
+        info: {},
+        doc: {}
+    }
+    worker TestNamedWorker {
+
+    }
+}
+
+@CustomAnnotation {
+    info: {},
+    doc: {}
+}
+client class TestClient {
+    @CustomAnnotation {
+        info: {},
+        doc: {}
+    }
+    int i;
+
+    @CustomAnnotation {
+        info: {},
+        doc: {}
+    }
+    remote function name() {
+
+    }
+
+    @CustomAnnotation {
+        info: {},
+        doc: {}
+    }
+    resource function accessor path() {
+
+    }
+
+}
+
+@CustomAnnotation {
+    info: {},
+    doc: {}
+}
+type TestError error<map<anydata>>;
+
+function testErrorVariable() {
+    @CustomAnnotation {
+        info: {},
+        doc: {}
+    }
+    error testErrorVar = error("Test Error");
+}
+
+[@CustomAnnotation {info: {}, doc: {}} int, int] [first, second] = [1, 2];
+
+// var fnPointer = function() {};
+
+// function testFunctionPointer() {
+//     @CustomAnnotation {
+//         info: {},
+//         doc: {}
+//     }
+//     fnPointer();
+// }
```

Comment:
commented?

Reply:
If this breaks the other test, that's okay to keep this comment. But, if this is because we are waiting for a fix to a test case and not breaking a test, it's okay to make it uncommented since we have anyway commenting the java test case.

Reply:
Ack